### PR TITLE
[2.4] Remove obsolete Red Hat Upstart and SuSE SysV init scripts

### PR DESCRIPTION
**Pull request by dgsga**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -350,13 +343,6 @@ AC_ARG_ENABLE(suse,
 	]
 )
 
-AC_ARG_ENABLE(suse-sysv,
-	[  --enable-suse-sysv      obsoleted ],[
-	echo "ERROR: --enable-suse-sysv is obsoleted. Use --enable-suse."
-	exit 1
-	]
-)
-
```
Comment (rdmark) [inline configure.ac:359]: The `--enable-suse` argument needs to be removed as well.

File: configure.ac [hunk 2]
```diff
@@ -327,13 +327,6 @@ AC_ARG_ENABLE(redhat,
 	]
 )
 
-AC_ARG_ENABLE(redhat-sysv,
-	[  --enable-redhat-sysv    obsoleted ],[
-	echo "ERROR: --enable-redhat-sysv is obsoleted. Use --enable-redhat."
-	exit 1
-	]
-)
```
Comment (rdmark) [inline configure.ac:335]: The `--enable-redhat` argument needs to be removed as well.

File: configure.ac (hunk 1 above)
Comment (dgsga) [reply configure.ac:359]: Done

File: configure.ac (hunk 2 above)
Comment (dgsga) [reply configure.ac:335]: Done

**Review by rdmark — APPROVED** [review]
Looks good!

But there is a merge conflict that you need to resolve. :)

**Pull request merged**